src/summary/cards/summary-card-row.component.tsx

```diff
@@ -6,9 +6,11 @@ export default function SummaryCardRow(props: SummaryCardRowProps) {
   return (
     <Link to={props.linkTo} className={`omrs-unstyled ${styles.row}`}>
       {props.children}
-      <svg className="omrs-icon" fill="var(--omrs-color-ink-low-contrast)">
-        <use xlinkHref="#omrs-icon-chevron-right" />
-      </svg>
+      {props.linkTo !== "" ? (
```

Comment:
When you want to do an if statement inside of jsx but there is no else statement, the normal way to do it is with `&&` instead of with a ternary expression. See [react docs](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) for more info.

Additionally, instead of doing a strict equality check for empty string, it would be better (and more common in javascript) to do a [truthiness check](https://developer.mozilla.org/en-US/docs/Glossary/Truthy). This is because we want the component to have good behavior when the user does not pass in the linkTo prop at all (which would make `linkTo` be `undefined`).

```suggestion
      {props.linkTo && (
```

Reply:
This is very usefull. Thanks for the information. I will use that from now.